### ISSUE TITLE
Treasury Report - Unblock testing of file-generation in staging environment for USDR_ADMIN users

### ISSUE THREAD

**Issue by as1729** [usdigitalresponse/cpf-reporter#348]
## Context
While some of the initial file generation pieces are being merged into staging, there is no way to easily test this feature piece-meal via the staging environment. The objective of this ticket is to be able to create a way for a `USDR_ADMIN` user to directly call each of the new lambda functions that are being added download the resulting files being generated.

## Current State
N/A

## Expected State

`USDR_ADMIN` users see the following buttons at their disposal on the homepage. Note: these buttons should not appear for any other user-role and will be discarded once the Treasury Report feature is fully built-out.

Form Field 1 - Accepts a long text containing payload from the user.
Button 1 - Kick-off file generation
Button 2 - Download Project 1A file
Button 3 - Download Project 1B file
Button 4 - Download Project 1C file
Button 5 - Download Subrecipient file

All the "Download ***" buttons should create a GET signed-url for the s3 object containing the project/subrecipient file for the current reporting period of the organization.

The "Kick-off file generation" button should send any payload that the user has added in "Form Field 1" to all the lambda functions.

## Implementation Details
- Add a Redwood Form
- All download buttons can follow the same pattern as the existing Upload file download button except the s3 key will be different.
- Add documentation for users on how to fill out the form and structure the payload

**Comment by ClaireValdivia** [comment]
Note for context - 295 & 296 & 320 are permanent solution for file generation. This issue (348) is a workaround to allow generation of files before those are done
